packages/repo-tools/src/commands/index.ts

```diff
@@ -133,6 +142,20 @@ function registerRepoCommand(program: Command) {
     .action(
       lazy(() => import('./repo/schema/openapi/fuzz').then(m => m.command)),
     );
+
+  openApiCommand
+    .command('check')
```

Comment:
Nit: It can be confusing because "check" sounds like "verify". Any thoughts on renaming to diff?

Reply:
I agree, I'll update